### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by drdavella**
It's not clear to me whether links should be to github or to pypi.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,7 +1,62 @@
...
-========
+==============
+pytest-astropy
+==============
 
-This is a meta package that pulls in dependencies on each of the new standalone
-pytest plugins that are used by the astropy core and affiliated packages. It
-is currently WIP.
+This is a meta-package that pulls in the dependencies that are used by
+`astropy`_ and `affiliated packages`_ for testing.
```
Comment (bsipocz) [inline README.rst]: "and many of the affiliated packages", not all use the astropy testing framework

**Comment by bsipocz** [comment]
links to github looks better for me, the links are usually only needed anyway when there is an issue...

**Review by bsipocz — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (pllim) [reply README.rst]: Some packages that use the test helpers are not even affiliated, so this sentence needs to be more inclusive.

File: README.rst [hunk 2]
```diff
@@ -1,7 +1,62 @@
...
+
+In either case, the plugin will automatically be registered for use with
+``pytest``.
+
+Development Status
+------------------
+
+Questions, bug reports, and feature requests can be submitted on `github`_.
+
+.. _github: https://github.com/astropy/pytest-remotedata
```
Comment (pllim) [inline README.rst]: Is this a copy-and-paste error?

**Pull request merged**